pkg/cli/debug/debug.go

```diff
@@ -499,6 +499,12 @@ func (o *DebugOptions) RunDebug() error {
 		return nil
 	}
 
+	msg, okayToDebug := safeToDebugPod(pod)
+	if !okayToDebug {
+		klog.Errorf(msg)
```

Comment:
This is a breaking change because we are no more allowing **any** pod is being debugged via debug command.